From 40aa494c026c67f089a4a84c600114473e9d7787 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ond=C5=99ej=20Lyson=C4=9Bk?= <olysonek@redhat.com>
Date: Mon, 7 May 2018 13:37:26 +0200
Subject: [PATCH] Remove superfluous call to get_input_value()
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

It is already called in the line above.
Discovered thanks to a Coverity Scan warning.

Signed-off-by: Ondřej Lysoněk <olysonek@redhat.com>
---
 prog/sensors/chips.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/prog/sensors/chips.c b/prog/sensors/chips.c
index 01c08fa8..99df8264 100644
--- a/prog/sensors/chips.c
+++ b/prog/sensors/chips.c
@@ -364,7 +364,6 @@ static void print_chip_temp(const sensors_chip_name *name,
 		sf = sensors_get_subfeature(name, feature,
 					    SENSORS_SUBFEATURE_TEMP_INPUT);
 		if (sf && get_input_value(name, sf, &val) == 0) {
-			get_input_value(name, sf, &val);
 			if (fahrenheit)
 				val = deg_ctof(val);
 			printf("%+6.1f%s  ", val, degstr);
-- 
2.14.3

